src/include/sound/volume_routines.rb

```diff
@@ -43,8 +45,8 @@ def setVolume(group, cardid, value)
         if Arch.sparc
```

Comment:
Ouch, SPARC is dead. We should rather remove that completely...

Reply:
I think half of sound is dead like that fancy script to copy sound blaster.